src/hist.jl

```diff
@@ -259,34 +272,42 @@ end
 append!{T,N}(h::AbstractHistogram{T,N}, vs::NTuple{N,AbstractVector}, wv::WeightVec) = append!(h, vs, values(wv))
 
 
+# Turn kwargs nbins into a type-stable tuple of integers:
+function _nbins_tuple{N}(vs::NTuple{N,AbstractVector}, nbins)
```

Comment:
That would be an elegant solution. Not on Julia v0.5, though, because it doesn't support broadcast for tuples (because size isn't defined for tuples):

    ERROR: MethodError: no method matching size(::Tuple{Int64,Int64})